spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -290,7 +285,16 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 				mutatedProps.put(provided, entry.getValue());
 			}
 		}
-		return builder.setProperties(mutatedProps).build(original.getStreamName());
+		return mutatedProps;
+	}
+
+	/**
+	 * Return a copy of a given app definition where short form parameters have been expanded to their long form
+	 * (amongst the whitelisted supported properties of the app) if applicable.
+	 */
+	/*default*/ StreamAppDefinition qualifyParameters(StreamAppDefinition original, Resource resource) {
```

Comment:
Also changing this method name (and the other) to `qualifyProperties` for consistency